src/main/java/racingcar/domain/Cars.java

```diff
@@ -0,0 +1,40 @@
+package racingcar.domain;
+
+import java.util.List;
+
+import static racingcar.validator.CarCountValidator.validateCarCount;
+
+public class Cars {
+    private final List<Car> cars;
+
+    private Cars(final List<Car> cars) {
+        this.cars = cars;
+    }
+
+    /* 기본 생성자 */
+    public static Cars createFromCarNames(final List<String> carNames) {
+        validateCarCount(carNames.size());
+        List<Car> cars = convertToCars(carNames);
+        return new Cars(cars);
+    }
+
+    /* 자동차 움직임 테스트에 사용되는 생성자
+       List<Car> 형식으로 생성하여 테스트 하기 위함 */
+    public static Cars createFromCarsList(final List<Car> cars) {
+        return new Cars(cars);
+    }
+
+    public List<Car> getCars() {
+        return cars;
```

Comment:
일급 컬렉션을 사용할 때 주의할 점이 있습니다.
일급 컬렉션은 컬렉션의 불변성을 보장하는데
이때 그냥 list를반환하면 외부에서 이 리스트에 값을 더하거나 제거할수있기 때문에 불변성이 깨지게됩니다
그래서
```suggestion
        return new ArrayList<>(cars);
```
이런식으로 세탁(?) 해서 사용하는게 좋을것 같습니다

Reply:
그렇군요..! 앞으로 주의해서 사용해보도록 할게요. 감사합니다!!